### ISSUE TITLE
Add a `getFieldError` helper

### ISSUE THREAD

**Issue by fakenickels** [Astrocoders/reform#100]
sometimes you just want to know if a field is errored or no and not care about the rest
so we could abstract this
```reason
        {getFieldState(Field(NumberOfFavoriteColors))
         |> (
           fun
           | Error(error) => Some(error)
           | _ => None
         )
         |> Belt.Option.getWithDefault(_, "")
         |> ReasonReact.string}
``` 

into

```reason
        {getFieldError(Field(NumberOfFavoriteColors))
         ->Belt.Option.getWithDefault("")
         ->ReasonReact.string}
```

**Comment by yurijean** [comment]
We can add this variation `getFieldErrorReactString(Field(NumberOfFavoriteColors))` besides `getFieldError(...)....`

**Comment by fakenickels** [comment]
hmm not sure if this could confuse beginners a bit, to already cast it to a React.string. Also we would already default the value to "" in order to do it. But indeed saves you from a lot of typing and duplication for this.

**Comment by LuizMoratelli** [comment]
getFieldError has been implemented in PR #102
